clients/src/main/java/org/apache/kafka/common/serialization/Serdes.java

```diff
@@ -80,6 +82,18 @@
         }
     }
 
+    static public final class BytesSerde implements Serde<Bytes> {
+        @Override
+        public Serializer<Bytes> serializer() {
+            return new BytesSerializer();
```

Comment:
I will add a public static object for each, and making the inner classes private.